src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -208,14 +208,14 @@ const formatLoadingTargetValue = (val) => {
     if (decimalPercentageRegex.test(val)) {
       return val
     } else {
-      return isNaN(well.value.loading_target_p1_plus_p2) ? 0 : well.value.loading_target_p1_plus_p2
+      return isNaN(well.value.loading_target_p1_plus_p2) ? 0 : well.loading_target_p1_plus_p2
```

Comment:
is well.value needed?

Reply:
Good spot, It is not needed as it is reactive now.